extensions/GlobalMessages/tests/GlobalMessages0.data.php

```diff
@@ -0,0 +1,970 @@
+// 21 --------------------------------------------------------------------------
+'closemyaccount-reactivate-intro' => 'You have previously requested that we close your account. You still have $1 days left until your account is closed. If you still wish to close your account, simply go back to browsing Fandom. However, if you would like to reactivate your account, please click the button below and follow the instructions in the email.
+
+Would you like to reactivate your account?',
+// 22 --------------------------------------------------------------------------
+'closemyaccount-scheduled-failed' => 'An error occurred while attempting to schedule this account to be closed. Please [[Special:CloseMyAccount|try again]] or [[Special:Contact|contact Fandom]] if the issue persists.',
+// 23 --------------------------------------------------------------------------
+'cnw-desc-all-ages' => 'Is this community intended for kids?',
+// 24 --------------------------------------------------------------------------
+'cnw-desc-creative' => 'Help people find your wiki with a superb description.',
+// 25 --------------------------------------------------------------------------
+'cnw-desc-default-lang' => 'Your community will be in $1',
+// 26 --------------------------------------------------------------------------
+'cnw-desc-headline' => 'What\'s your wiki about?',
+// 27 --------------------------------------------------------------------------
+'cnw-desc-placeholder' => 'Make it good! Your text will appear on the main page of your wiki.',
+// 28 --------------------------------------------------------------------------
+'cnw-desc-tip-all-ages' => 'Is this about a topic that children are interested in? In order to help us comply with US law we keep track of communities about topics that directly appeal to children 12 and under.',
+// 29 --------------------------------------------------------------------------
+'cnw-desc-tip1-creative' => 'Use this space to tell people why this community matters and the reason you created it.',
+// 30 --------------------------------------------------------------------------
+'cnw-name-wiki-domain-label' => 'Give your wiki an address',
+// 31 --------------------------------------------------------------------------
+'cnw-name-wiki-headline' => 'Start a wiki',
+// 32 --------------------------------------------------------------------------
+'cnw-name-wiki-label' => 'Name your wiki',
+// 33 --------------------------------------------------------------------------
+'cnw-userauth-marketing-body' => 'You need an account to create a wiki on Fandom. It only takes a minute to sign up!',
+// 34 --------------------------------------------------------------------------
+'cnw-welcome-instruction1' => 'Click the button below to start adding pages to your wiki.',
+// 35 --------------------------------------------------------------------------
+'contact' => 'Contact Fandom',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Contact Fandom" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)